### PR TITLE
faq/tidb.md: update wording about transaction size limit

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What is changed, added or deleted? 

Update transaction size description

### What is the related PR or file link(s)? 

- Reference link(s): https://github.com/pingcap/docs-cn/pull/2255
- This PR is to align with:
- [x] N/A (not applicable)

### CODE REVIEW

File: faq/tidb.md [hunk 1]
```diff
@@ -842,14 +842,15 @@ TiDB 读流量可以通过增加 TiDB server 进行扩展，总读容量无限
 
 #### 4.3.3 Transaction too large 是什么原因，怎么解决？
 
-由于分布式事务要做两阶段提交，并且底层还需要做 Raft 复制，如果一个事务非常大，会使得提交过程非常慢，并且会卡住下面的 Raft 复制流程。为了避免系统出现被卡住的情况，我们对事务的大小做了限制：
+由于分布式事务要做两阶段提交，并且底层还需要做 Raft 复制，如果一个事务非常大，提交过程非常慢，事务写冲突概率会增加，并且事务失败回滚会导致不必要的性能开销。我们对事务的做了限制：
```
Comment (TomShawn) [inline faq/tidb.md]:
```suggestion
分布式事务要做两阶段提交，而且底层还需要做 Raft 复制。如果一个事务非常大，提交过程会非常慢，事务写冲突概率会增加，而且事务失败后回滚会导致不必要的性能开销。因此我们对事务的做了以下限制：
```

File: faq/tidb.md [hunk 2]
```diff
@@ -842,14 +842,15 @@ TiDB 读流量可以通过增加 TiDB server 进行扩展，总读容量无限
...
 
 - 单个事务包含的 SQL 语句不超过 5000 条（默认）
 - 单条 KV entry 不超过 6MB
-- KV entry 的总大小不超过 100MB
 
 在 Google 的 Cloud Spanner 上面，也有类似的[限制](https://cloud.google.com/spanner/docs/limits)。
 
+默认还设置了 KV entry 的总大小不超过 100MB，如果业务需要使用大事务，可能修改配置文件中的配置项 txn-total-size-limit 调整，最大可以修改到 10G。实际的大小限制还受机器的物理内存影响。
```
Comment (TomShawn) [inline faq/tidb.md]:
```suggestion
还默认限制 KV entry 的总大小不超过 100MB。如果业务需要使用大事务，可以修改配置文件中的 `txn-total-size-limit` 配置项进行调整，最大可以修改到 10G。实际的大小限制还受机器的物理内存影响。
```

**Comment by tiancaiamao** [comment]
PTAL @coocood @TomShawn 

See also https://github.com/pingcap/docs/pull/1876

**Comment by TomShawn** [comment]
LGTM

**Comment by tiancaiamao** [comment]
Tiny update, remove the description about 5000 statement limitation inside a transaction.
@coocood @TomShawn

**Comment by coocood** [comment]
LGTM

File: faq/tidb.md [hunk 3]
```diff
@@ -842,11 +838,9 @@ TiDB 读流量可以通过增加 TiDB server 进行扩展，总读容量无限
 
 #### 4.3.3 Transaction too large 是什么原因，怎么解决？
 
-由于分布式事务要做两阶段提交，并且底层还需要做 Raft 复制，如果一个事务非常大，会使得提交过程非常慢，并且会卡住下面的 Raft 复制流程。为了避免系统出现被卡住的情况，我们对事务的大小做了限制：
+TiDB 限制了单条 KV entry 不超过 6MB。
 
-- 单个事务包含的 SQL 语句不超过 5000 条（默认）
-- 单条 KV entry 不超过 6MB
-- KV entry 的总大小不超过 100MB
+分布式事务要做两阶段提交，而且底层还需要做 Raft 复制。如果一个事务非常大，提交过程会非常慢，事务写冲突概率会增加，而且事务失败后回滚会导致不必要的性能开销。所以我们设置了 KV entry 的总大小默认值不超过 100MB。如果业务需要使用大事务，可以修改配置文件中的 `txn-total-size-limit` 配置项进行调整，最大可以修改到 10G。实际的大小限制还受机器的物理内存影响。
```
Comment (TomShawn) [inline faq/tidb.md]:
```suggestion
分布式事务要做两阶段提交，而且底层还需要做 Raft 复制。如果一个事务非常大，提交过程会非常慢，事务写冲突概率会增加，而且事务失败后回滚会导致不必要的性能开销。所以我们设置了 key-value entry 的总大小默认不超过 100MB。如果业务需要使用大事务，可以修改配置文件中的 `txn-total-size-limit` 配置项进行调整，最大可以修改到 10G。实际的大小限制还受机器的物理内存影响。
```

**Comment by tiancaiamao** [comment]
PTAL @TomShawn

**Pull request merged**